Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/SeedingAlgorithm.hpp

```diff
@@ -63,7 +63,7 @@ class SeedingAlgorithm final : public IAlgorithm {
     std::vector<std::pair<int, int>> zBinNeighborsBottom;
     // number of phiBin neighbors at each side of the current bin that will be
     // used to search for SPs
-    int numPhiNeighbors = 0;
+    int numPhiNeighbors = 1;
```

Comment:
What happens in the grid iteration if the local position happens to be at the edge of the phi axis? Does the iteration wrap around at -pi <-> pi or does this never occur?

Reply:
that should be handled automatically, since the phi axes is a `detail::AxisBoundaryType::Closed`. That means (according to documentation in https://github.com/acts-project/acts/blob/main/docs/core/misc/grid_axis.md?plain=1#L32) that "out-of-bounds lookups wrap-around to the other side of the axis"

Reply:
But indeed we can add some unit tests

Reply:
Added a test dedicated to that

Reply:
Nice, thanks! I was just worried that it might not do that when you look up bins by any kind of bin indices instead of from a position, but fair enough